bitcoin/src/address.rs

```diff
@@ -554,8 +564,54 @@ impl<'a> fmt::Display for AddressEncoding<'a> {
     }
 }
 
+/// Marker of status of address's network validation.
+pub trait NetworkValidation {}
```

Comment:
I think this should be sealed.

Reply:
Yes, makes sense. Done in 0e448ae3398cd4486855693f524c7e0e40f7f902.